packages/compare-images/python/itkwasm-compare-images/itkwasm_compare_images/compare_double_images_async.py

```diff
@@ -12,6 +12,7 @@ async def compare_double_images_async(
     test_image: Image,
     baseline_images: List[Image] = [],
     difference_threshold: float = 0,
+    spatial_tolerance: float = 1e-06,
```

Comment:
again

Reply:
also missed out on updating the wasm binary